app/move/middleware.js

```diff
@@ -21,7 +21,7 @@ module.exports = {
 
     try {
       const move = await req.services.move.getByIdWithEvents(moveIdWithEvents)
-      await populateResources(move.timeline_events, req)
```

Comment:
`populateResources` is no longer asynchronous :+1:

Reply:
Nope. It may well have to be in the future but for now I've removed the extra API calls. They appear to be what was breaking production. I think we need to collect some data about instances where the loops never ends before we re-implement this code.